README.md

```diff
@@ -13,6 +13,13 @@ router.route()
 
 ```
 
+## Accessing server span context
+Because Vert.x is event loop based, thread local implementations of span source do not work.
```

Comment:
As active span source will soon disappear might be easier to just leave out this sentence and just say how to get the context as below?

Reply:
I would leave it to make people more aware, for case they would like to use any span Manager for passing the spans internally.

Reply:
I mean TLS based span storage is not going away. It's just a name change.